src/mavlink-router/ulog.cpp

```diff
@@ -150,6 +153,15 @@ int ULog::write_msg(const struct buffer *buffer)
         return buffer->len;
     }
 
+    uint16_t crc_msg = payload[payload_len] | (payload[payload_len + 1] << 8);
```

Comment:
crc needs to be checked before _handle_auto_start_stop

Reply:
True